wearApp/src/main/java/dev/johnoreilly/confetti/wear/navigation/WearAppComponent.kt

```diff
@@ -80,25 +81,16 @@ class DefaultWearAppComponent(
     private val networkRepository: NetworkRepository by inject()
     private val dataRequestRepository: DataRequestRepository by inject()
 
-    val conferenceDataFlow = phoneSettingsSync.conferenceFlow.map {
-        if (it.isNotBlank()) {
-            repository.conferenceData(it, FetchPolicy.CacheFirst).data
```

Comment:
This is really the point of this, want to be able to avoid an Apollo call before knowing where to do.